src/libs/PolicyUtils.ts

```diff
@@ -158,7 +158,7 @@ function getIneligibleInvitees(policyMembers: OnyxEntry<PolicyMembers>, personal
 /**
  * Gets the tag from policy tags, defaults to the first if no key is provided.
  */
-function getTag(policyTags: OnyxEntry<PolicyTags>, tagKey?: keyof typeof policyTags): PolicyTag | undefined | EmptyObject {
+function getTag(policyTags: OnyxCollection<PolicyTags>, tagKey?: keyof typeof policyTags): PolicyTags | undefined | EmptyObject {
```

Comment:
Also unsure about this, can you explain?

Reply:
I mean, types are not right. Any doubts??

Reply:
I meant that I'm not sure how it is related to the changes in this PR?

Reply:
I think I'd prefer if unrelated changes are reverted so that we keep things simple here! If you've seen typing issues you can raise them on Slack.